qemu/tests/nvdimm.py

```diff
@@ -1,7 +1,10 @@
 import logging
+import os
```

Comment:
Could you please explain why?
Other parts are updated, please help review again, thanks!